config/mainnet/rollup.json

```
@@ -18,8 +18,8 @@
     }
   },
   "block_time": 2,
-  "max_proposer_drift": 600,
-  "proposer_window_size": 3600,
+  "max_sequencer_drift": 600,
+  "sequencer_window_size": 3600,
```

Looks like it needs to be renamed.
```suggestion
  "seq_window_size": 3600,
```